src/main/java/org/jpeek/skeleton/OpsOf.java

```diff
@@ -89,15 +96,22 @@ public void visitMethodInsn(final int opcode,
         final String owner, final String mtd,
         final String dsc, final boolean itf) {
         super.visitMethodInsn(opcode, owner, mtd, dsc, itf);
-        final String[] args = dsc.substring(dsc.indexOf('(') + 1, dsc.indexOf(')')).split(";");
+        final Iterable<Text> args = new Split(
+            new Sub(dsc, dsc.indexOf('(') + 1, dsc.indexOf(')')),
+            ";"
+        );
         this.target.strict(1).addIf("ops").add("op");
         this.target
             .attr("code", "call")
             .add("name")
             .set(owner.replace("/", ".").concat(".").concat(mtd))
             .up().add("args");
-        for (final String arg : args) {
-            this.target.add("arg").attr("type", arg).set("?").up();
+        for (final Text arg : args) {
+            try {
+                this.target.add("arg").attr("type", arg.asString()).set("?").up();
+            } catch (final Exception ex) {
```

Comment:
@HDouss You don't need to catch exception here.
What you can do is: 
1. Do not use `arg.asString()`, just use `arg`, so that way `toString()` will be called and `asString `will called by `toString()` by Text implementation.

2. Alternatively you can wrap `arg `into `UncheckedText` and call `asString()`

Reply:
@fanifieiev makes sens. Fixed!